tests/settings.py

```diff
@@ -10,3 +10,4 @@
 ELASTICSEARCH_DSL = {
     "default": {"hosts": os.environ.get("ELASTICSEARCH_HOST", "localhost:9201")}
 }
+DATE_FORMAT = "%Y.%m.%d"
```

Comment:
Perhaps we should omit this from the settings and test that the default is respected and that it is override-able.

Reply:
Also, this should be namespaced, e.g. `ELASTICSEARCH_METRICS_DATE_FORMAT`.